### DISCUSSION TITLE
DBConnection when using Azure User Managed Identity

### DISCUSSION THREAD

**Discussion by EclypseDesigns** [elsa-workflows/elsa-core#3774] [Q&A] [answered]
Hi all :) Been going around in circles over this one. We want to leverage EntityFrameworkPersistence against a provisioned Azure SQL database.

I've setup a managed identity and attempted to construct the necessary 'magic' connection string but no amount of cussing has it working. 

Is there an example anywhere i can check out which possibly has a workaround for this scenario ? Ideall something which leverages 

`new DefaultAzureCredential(new DefaultAzureCredentialOptions { ManagedIdentityClientId = "9999" });`

Many thanks 

B

**Comment by EclypseDesigns** [comment, marked as answer]
Ignore me. Being dense. This has been answered https://github.com/elsa-workflows/elsa-core/discussions/3074

tested it and working spot on :)

**Reply by sfmskywalker** [reply]
Glad to see you figured it out. Thanks for the update!
